### DISCUSSION TITLE
AGW mme.log size and rotation control how to

### DISCUSSION THREAD

**Discussion by pkapatsos-blinq** [magma/magma#8940] [Q&A]
Hi where is the MME.LOG size set or controlled via please. I see it rotates if a reboot occurs but cant find where to configure the size to rotate . With real world this file can become unmanageable  if we don't set a value.
I see under cat /etc/logrotate.d there is an OAI ref but NO SIZE SETTING  , also see the syslog for magma which is working correctly @ 50M.
**Where is the MME.LOG file size being controlled or set and how to change?
Can we please get a doc or page on controlling these log files over all on the AGW ?please.**

/oai
/var/log/mme.log
{
  rotate 20
  daily
  missingok
  notifempty
  compress
  copytruncate
}
cat /etc/logrotate.d/rsyslog.magma
/var/log/syslog
{
        rotate 7
        daily
        missingok
        notifempty
        delaycompress
        compress
        maxsize 50M
        postrotate
                invoke-rc.d rsyslog restart > /dev/null
        endscript
}

**Comment by joary** [comment]
Hi @pkapatsos-blinq , this is a logrotate specific configuration, you can use `maxsize` and `minsize` options [see documentation](https://manpages.debian.org/jessie/logrotate/logrotate.8.en.html). Also [an example on how to configure it](https://askubuntu.com/questions/936878/how-to-rotate-a-log-file-in-ubuntu-on-size-basis-hourly)

**Comment by pkapatsos-blinq** [comment]
@joary Ok so you have no size for mme.log then set correct ?

**Comment by hfvilca** [comment]
Hi @pkapatsos-blinq , have you found a way to rotate the mme logs?

**Comment by voisey** [comment]
For what it's worth, I tried following the instructed posted by @joary and was also unable to restrict the size of the file. I tried doing this in both `/etc/logrotate.d/oai` and `etc/logrotate.d/rsyslog`, yet after restarting MME mme.log still exceeded `maxsize`. Perhaps I have missed a crucial point though. Has someone been able to do this successfully?
